src/app/workflow/workflow.component.html

```diff
@@ -190,7 +190,15 @@ <h3>
       </mat-tab>
     </mat-tab-group>
   </div>
-  <div *ngIf="publicPage" class="mt-2" id='discourse-comments'></div>
+
+  <div *ngIf="publicPage" class="mt-2 mr-3">
+    <div *ngIf="workflow?.topicId !== null else noTopicId" id='discourse-comments'></div>
```

Comment:
Can the topicId be an empty string if it's not set or will it be null? If there's any doubt, you should change this to `*ngIf="workflow?topicId else noTopicId`.

Reply:
It should either be null or set, but if that handles both cases I'll change it.